apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -82,11 +87,27 @@ const TutorialExplorer = React.createClass({
     });
   },
 
-  filterTutorials(filters, hideFilters) {
+  /**
+   * Called when the sort order is changed via dropdown.
+   *
+   * @param {string} value - The new sort order.
```

Comment:
Should this actually be `@param {SortBy} value - The new sort order` now that you have an enum?